recipes/xmitgcm/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xmitgcm" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4378d4982a86786daaf49690ad3d144a1296c2fa385ed86f03394e5b655ea8f9
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray
+    - dask
+    - numpy
+    - future
+    - docrep
+
+test:
+  imports:
+    - xmitgcm
+
+about:
+  home: https://github.com/xgcm/xmitgcm
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: Read MITgcm mds binary files into xarray
+  dev_url: https://github.com/xgcm/xmitgcm
+  doc_url: https://xmitgcm.readthedocs.io/
+
+extra:
+  recipe-maintainers:
+    - raphaeldussin
```

Comment:
Have all co-maintainers agreed to be included? If you don't have their approval, please ping them in this Pull Request.

Reply:
yes

https://github.com/xgcm/xmitgcm/issues/75